dts/msm8226/msm8926-huawei-g6-l11.dts

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+#include <skeleton.dtsi>
+
+/ {
+	model = "Qualcomm MSM 8926 CDP G6-L11";
+	compatible = "huawei,g6-l11", "qcom,msm8926-cdp", "qcom,msm8926", "lk2nd,device";
+
+	qcom,msm-id = <200 0x10000>;
+	qcom,board-id = <0 0>;
```

Comment:
Hm, this doesn't seem to match https://github.com/LineageOS/android_kernel_huawei_msm8226/blob/cm-12.1/arch/arm/boot/dts/msm8926-g6_l11-vb.dts#L25 or am I looking at the wrong device?

Reply:
hi i have this device, helping mirsal testing... 
Indicate me how can i help..
The phone name i Huawei Ascend G6 l11 LTE
 
[440] Device model: Qualcomm MSM 8926 CDP G6-L11
[440] Found an appended flattened device tree (Qualcomm MSM 8926 CDP G6-L11 - 200 0 0 0x10000) 
[450] Device tree exact match the board: <200 0 65536 0x0> == <200 0 0 0x10001>
[460] LK would match our own DTB. Cool!
from the lk.log of lk2nd fixed by mirsal 

the The difference between MSM8926 and MSM8926 is the support of the LTE in the MSM8926

Reply:
@stephan-gh thanks a lot for your review. it it might be a different device, or maybe a different revision. the device this was tested on has the following in the dtb bundled with stock android images:

```
	model = "Qualcomm MSM 8926 CDP g6-l11-va";
	compatible = "qcom,msm8926-cdp", "qcom,msm8926", "qcom,cdp";
	qcom,msm-id = <0xc8 0x1f41 0x00>;
```
which is also different

lineageos has:

```
model = "Qualcomm MSM 8926 CDP g6-l11-va";
compatible = "qcom,msm8926-cdp", "qcom,msm8926", "qcom,cdp";
qcom,board-id = <8001 0>;
```
in https://github.com/LineageOS/android_kernel_huawei_msm8226/blob/cm-12.1/arch/arm/boot/dts/msm8926-g6_l11-va.dts

the stock bootloader matches the first dtb fragment appended to the flashed lk2nd image no matter what, so I used lk2nd logs along with trial and error in order to find matching values.

Any pointer to a better way would be greatly appreciated!

Reply:
If we use the exact values reported by lk2nd (`qcom,msm-id = <200 0>;` and `qcom,board-id = <0 0x10001>;`) then the result is the following:

```
[380] ### Ohai, this is lk2nd (or lk1st?) ###
[380] Board: platform: 200, foundry: 0x1, platform_version: 0x10001, platform_hw: 0x0, platform_subtype: 0x0, target: 0x0, baseband: 0x0, platform_hlos_subtype: 0x0
[400] pmic_info[0]: type: 0x10004, version: 0x20002, target: 0x20204
[400] Command line from primary bootloader: lk2nd androidboot.emmc=true androidboot.serialno=a9a2c30 androidboot.huawei_swtype=normal androidboot.huawei_bootmode=boot androidboot.localproppath=vodafone/default usb.pid.index=25 usb.serial=2008edb117cd androidboot.baseband=msm mdss_mdp.panel=1:dsi:0:qcom,mdss_dsi_tianma_otm9605a_4p5_qhd_video
[430] Device model: Huawei Ascend G6-L11
[440] Found an appended flattened device tree (Huawei Ascend G6-L11 - 200 0 65537 0x0)
[440] Device tree exact match the board: <200 0 0 0x10001> == <200 0 0 0x10001>
[450] Weird DTB selected by bootloader. Need to override. Sad.
```

Reply:
```
qcom,msm-id = <200 0>, ...
qcom,board-id = <8001 0>;
```

is mostly equivalent to

```
qcom,msm-id = <0xc8 0x1f41 0x00> = <200 8001 0>;
```

So the two examples you show are the same. And one of them (preferably the first) is what I'd expect to have in lk2nd. :)

Reply:
Thanks, yes, that was my first guess but then I got confused by lk2nd logs. Fixing.